extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php

```diff
@@ -570,16 +568,15 @@ function axWFactorySaveVariable() {
 				 * catch parse errors
 				 */
 				ob_start();
-				if( eval( $tEval ) === FALSE ) {
+				if ( eval( $tEval ) === FALSE ) {
 					$error++;
 					$return = Wikia::errormsg( "Syntax error, value is not valid PHP structure. Variable not saved." );
-				}
-				else {
+				} else {
 					$cv_value = $__var_value;
 					/**
 					 * now check if it's actually array when we want array)
 					 */
-					if( in_array( $cv_variable_type, array( "array", "struct", "hash" ) ) ) {
+					if( in_array( $cv_variable_type, [ "array", "struct", "hash" ] ) ) {
 						if( is_array( $cv_value ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)